Omnibus.md

```diff
@@ -3052,13 +3075,15 @@ be moving their arms (e.g., parrying, so long as they don't step backwards).
 
 __Uses:__ Unlimited, one at a time
 
-Having this spell makes the spellcaster harder to scalp. The next time the
-spellcaster is scalped it will take 200 extra blows to successfully scalp them.
-If scalped for only 200 blows, instead of the full 400 blows, the spellcaster
-must inform the scalper that "The job is not yet done." A spellcaster can only
-be under the effect of one Heartiness spell at a time. A use is considered to be
-over whenever the spellcaster receives at least 200 scalping blows, but is in
-effect until they are either fully scalped or raised.
+Having this spell makes [it harder to destroy the spellcaster's body]{.new}. 
+The next time the spellcaster['s body is destroyed]{.new} it will take 200 
+extra blows to successfully [destroy their body]{.new}. If struck for only 
+200 blows, instead of the full 400 blows, the spellcaster must inform the 
+[individual(s) destroying their body]{.new} that "The job is not yet done." 
+A spellcaster can only be under the effect of one Heartiness spell at a time. 
+A use is considered to be over whenever the spellcaster receives at least 
```

Comment:
Out of scope, but I want to rant (and remember to write a proposal for next year) about how awful this spell description is.  A use is over when [EVENT] happens, but it's still in effect until [OTHER EVENT] happens.  WHEN DOES IT END?

HOW DO YOU (RE-)CAST THIS SPELL?  WHY DOES IT EVER END?

Reply:
This is what happens when you try to make spells that are just skills or abilities.

Reply:
I already made this joke to Pi, but I'm gonna leave it here as well.

If you only have heartiness, you have spells but can't cast any.  Does this mean you are not a spellcaster, but merely a spellhaver?